src/eachrow.jl

```diff
@@ -95,24 +95,25 @@ end
 """
     @eachrow(d, body)
 
-Act on each row of a data frame, similar to
+Act on each row of a data frame, producing a new dataframe.
+Similar to
 
 ```
-for row in eachrow(df)
+for row in eachrow(copy(df))
```

Comment:
I wouldn't use `copy` here, as it doesn't make a difference if you don't mutate `row`, which the example doesn't show.

Reply:
changed.

Reply:
It's still there.

Reply:
I will keep this in in-light of performance gotcha that `@eachrow` always copies.